### PR TITLE
Assist Panel: Button and showing assist panel

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/649
[Problem] Primary and assist panel is not implemented
[Solution] Assist panel is similar to Drawer widget

### CODE REVIEW

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<body>
+	<div class="ui-page" id="assist-panel-page">
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Assist Panel
+			</h2>
+		</header>
+		<div class="ui-content">
+			Empty content
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/navigationelements/assist-panel.html]: not exist in guide

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	</script>
+	<script src="../../lib/tau/wearable/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="assist-panel-page">
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Assist Panel
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/navigationelements/assist-panel.html]: not exist in guide

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+			</h2>
+		</header>
+		<div class="ui-content">
+			Empty content
+		</div>
+		<div class="ui-assist-panel" id="assistPanel" data-drag-edge="50" data-position="down">
+			<ul class="ui-listview">
+				<li>
+					<a href="#" data-rel="back">
+						Assit Panel Item 1
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/navigationelements/assist-panel.html:27]: lease make sure that opacity of assist panel is correct

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+	<div class="ui-page" id="assist-panel-page">
+		<header class="ui-header">
+			<h2 class="ui-title">
+				Assist Panel
+			</h2>
+		</header>
+		<div class="ui-content">
+			Empty content
+		</div>
+		<div class="ui-assist-panel" id="assistPanel" data-drag-edge="50" data-position="down">
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/navigationelements/assist-panel.html]: as we discussed data-position should be default for down

File: examples/wearable/UIComponents/contents/navigationelements/index.html [hunk 5]
```diff
@@ -41,8 +41,8 @@ <h2 class="ui-title">
 					</a>
 				</li>
 				<li>
-					<a href="drawer.html">
-						Drawer
+					<a href="assist-panel.html">
+						Assist Panel
```
Comment (lslachciak) [inline examples/wearable/UIComponents/contents/navigationelements/index.html:45]: to consider: in guides it says Primary & Assist Panel

File: src/js/core/widget/core/Drawer.js [hunk 6]
```diff
@@ -27,6 +27,8 @@
  *
  * - left (left position, default)
  * - right (right position)
+ * @since 1.2
+ * - down (bottom position)
```
Comment (lslachciak) [inline src/js/core/widget/core/Drawer.js:31]: how about -up ?

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/wearable/UIComponents/contents/navigationelements/assist-panel.html]: Done

Comment (TomaszLukawskiSam) [reply examples/wearable/UIComponents/contents/navigationelements/assist-panel.html]: Removed

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/wearable/UIComponents/contents/navigationelements/assist-panel.html]: Removed

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/wearable/UIComponents/contents/navigationelements/assist-panel.html:27]: Done

File: examples/wearable/UIComponents/contents/navigationelements/assist-panel.html (hunk 4 above)
Comment (TomaszLukawskiSam) [reply examples/wearable/UIComponents/contents/navigationelements/assist-panel.html]: Done. Removed.

File: examples/wearable/UIComponents/contents/navigationelements/index.html (hunk 5 above)
Comment (TomaszLukawskiSam) [reply examples/wearable/UIComponents/contents/navigationelements/index.html:45]: What means "Primary"? Is this means Primary Panel and Assist Panel. We can change this when we will have more guidelines.

File: src/js/core/widget/core/Drawer.js (hunk 6 above)
Comment (TomaszLukawskiSam) [reply src/js/core/widget/core/Drawer.js:31]: Done. The `up` has been added.

File: examples/wearable/UIComponents/contents/navigationelements/index.html (hunk 5 above)
Comment (lslachciak) [reply examples/wearable/UIComponents/contents/navigationelements/index.html:45]: ok

**Review by lslachciak — APPROVED** [review]

**Pull request merged**